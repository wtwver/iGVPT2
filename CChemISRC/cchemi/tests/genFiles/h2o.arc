

                     SUMMARY OF PM3 CALCULATION, Site No: 1079

                                                       MOPAC2012 (Version: 15.146L)
                                                       Fri Dec 11 08:18:33 2015
                                                       No. of days left = 166

           Empirical Formula: H2 O  =     3 atoms

PM3 XYZ CHARGE=0 Singlet AUX

 TEST


     GRADIENTS WERE INITIALLY ACCEPTABLY SMALL                
     SCF FIELD WAS ACHIEVED                                   

          HEAT OF FORMATION       =        -53.43282 KCAL/MOL =    -223.56294 KJ/MOL
          TOTAL ENERGY            =       -324.90736 EV
          ELECTRONIC ENERGY       =       -474.29001 EV
          CORE-CORE REPULSION     =        149.38265 EV
          GRADIENT NORM           =          0.11179
          DIPOLE                  =          1.73857 DEBYE    POINT GROUP:       C2v 
          NO. OF FILLED LEVELS    =          4
          IONIZATION POTENTIAL    =         12.316509 EV
          HOMO LUMO ENERGIES (EV) =        -12.317  4.060
          MOLECULAR WEIGHT        =         18.015
          COSMO AREA              =         42.47 SQUARE ANGSTROMS
          COSMO VOLUME            =         25.19 CUBIC ANGSTROMS

          MOLECULAR DIMENSIONS (Angstroms)

            Atom       Atom       Distance
            H     3    H     2     1.53589
            H     2    O     1     0.56094
            O     1    H     2     0.00000
          SCF CALCULATIONS        =          1
          WALL-CLOCK TIME         =          0.014 SECONDS
          COMPUTATION TIME        =          0.016 SECONDS


          FINAL GEOMETRY OBTAINED
PM3 XYZ CHARGE=0 Singlet AUX

 TEST
  O    -0.00001000 +1  -0.38096200 +1   0.00000000 +1
  H     0.76796600 +1   0.17999600 +1   0.00000000 +1
  H    -0.76792600 +1   0.17996400 +1   0.00000000 +1
 
