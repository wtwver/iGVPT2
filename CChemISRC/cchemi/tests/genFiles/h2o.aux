 START OF MOPAC FILE
 ####################################
 #                                  #
 #       Start of Input data        #
 #                                  #
 ####################################
 MOPAC_VERSION=MOPAC2012.15.146L
 DATE="Fri Dec 11 08:34:06 2015"
 METHOD=PM3
 TITLE=""
 KEYWORDS=" PM3 FORCE CHARGE=0 SINGLET AUX NOREOR"
 COMMENTS="TEST"
 ATOM_EL[0003]=
  O  H  H 
 ATOM_CORE[0003]=
  6  1  1 
 ATOM_X:ANGSTROMS[0009]=
   -0.0000   -0.3810    0.0000
    0.7680    0.1800    0.0000
   -0.7679    0.1800    0.0000
 AO_ATOMINDEX[0006]=
    1    1    1    1    2    3
 ATOM_SYMTYPE[0006]=
  S PX PY PZ  S  S 
 AO_ZETA[0006]=
  3.7965  2.3894  2.3894  2.3894  0.9678  0.9678
 ATOM_PQN[0006]=
 2 2 2 2 1 1
 NUM_ELECTRONS=0008
 EMPIRICAL_FORMULA="H2 O  =     3 atoms"
 ####################################
 #                                  #
 #      Geometry optimization       #
 #                                  #
 ####################################
 ####################################
 #                                  #
 #        Final SCF results         #
 #                                  #
 ####################################
 HEAT_OF_FORMATION:KCAL/MOL=-0.534328D+02
 ENERGY_ELECTRONIC:EV=-0.474290D+03
 ENERGY_NUCLEAR:EV=+0.149383D+03
 GRADIENT_NORM:KCAL/MOL/ANGSTROM=+0.143120D+00
 POINT_GROUP=Cs  
 DIPOLE:DEBYE=+0.173865D+01
 DIP_VEC:DEBYE[3]= +0.21346D-04 +0.17386D+01 -0.13551D-14
 IONIZATION_POTENTIAL:EV=+0.123164D+02
 SPIN_COMPONENT=+0.000000D+00
 TOTAL_SPIN=+0.000000D+00
 NUMBER_SCF_CYCLES=2
 NUM_ALPHA_ELECTRONS=0
 NUM_BETA_ELECTRONS=0
 CPU_TIME:SEC=+0.781250D-02
 MOLECULAR_WEIGHT:AMU=+0.180152D+02
 TOTAL_ENERGY:EV=-0.324907D+03
 ATOM_X_OPT:ANGSTROMS[0009]=
   -0.0000   -0.3810    0.0000
    0.7680    0.1800    0.0000
   -0.7679    0.1800    0.0000
 ATOM_CHARGES[0003]=
 -0.35860 +0.17930 +0.17930
 GRADIENTS:KCAL/MOL/ANGSTROM[0009]=
  -0.0641  -0.0286   0.0000   0.1095   0.0377   0.0000  -0.0454  -0.0092   0.0000
 OVERLAP_MATRIX[000021]=
 #  Lower half triangle only
   1.0000   0.0000   1.0000   0.0000   0.0000   1.0000   0.0000   0.0000   0.0000   1.0000
   0.2916   0.2588   0.1890   0.0000   1.0000   0.2917  -0.2588   0.1890   0.0000   0.3881
   1.0000
 SET_OF_MOS=       1       6
 EIGENVECTORS[000036]=
   0.8779  -0.0000   0.1049   0.0000   0.3303   0.3304   0.0000   0.7680  -0.0000   0.0000
   0.4529  -0.4529  -0.3330   0.0000   0.8347  -0.0000   0.3101   0.3100  -0.0000  -0.0000
   0.0000   1.0000  -0.0000   0.0000  -0.3440  -0.0002  -0.5406  -0.0000   0.5430   0.5428
  -0.0001   0.6405  -0.0001  -0.0000  -0.5429   0.5432
 TOTAL_DENSITY_MATRIX[000021]=
 #  Lower half triangle only
   1.7634   0.0000   1.1796  -0.3719  -0.0000   1.4156   0.0000   0.0000   0.0000   2.0000
   0.3735   0.6956   0.5869   0.0000   0.8207   0.3735  -0.6956   0.5869   0.0000   0.0003
   0.8207
 M.O.SYMMETRY_LABELS[0006]=
   1a'     2a'     3a'     1a''    4a'     5a'  
 EIGENVALUES[0006]=
  -36.825  -17.581  -14.523  -12.316    4.060    5.332
 MOLECULAR_ORBITAL_OCCUPANCIES[00006]=
 2.0000 2.0000 2.0000 2.0000 0.0000 0.0000
 ####################################
 #                                  #
 #      Normal mode analysis        #
 #                                  #
 ####################################
 ORIENTATION_ATOM_X:ANGSTROMS[0009]=
   -0.0000   -0.3810    0.0000
    0.7680    0.1800    0.0000
   -0.7679    0.1800    0.0000
 ROTAT_CONSTS:CM(-1)[3]=    29.925994    14.180376     9.621328
 PRI_MOM_OF_I:10**(-40)*GRAM-CM**2[3]     0.935400     1.974050     2.909450
 ZERO_POINT_ENERGY:KCAL/MOL=    13.721354
 VIB._FREQ:CM(-1)[0009]=
  1741.97  3867.83  3988.42 -5775.26  7420.35     9.53 -2044.29 -7132.56  5776.42
 NORMAL_MODES[000081]=
  -0.0000  -0.2830   0.0011  -0.3771   0.5637  -0.0031   0.3771   0.5637  -0.0011  -0.2755
  -0.0004  -0.0007   0.5503   0.4018   0.0000   0.5475  -0.4001   0.0030  -0.0006   0.1784
  -0.0007  -0.5969  -0.3544  -0.0003   0.5994  -0.3563   0.0030   0.0002  -0.0014  -0.0870
  -0.0019   0.0024   0.7044  -0.0019   0.0024   0.7044  -0.1835   0.0000  -0.0005  -0.4395
   0.5386  -0.0011  -0.4394  -0.5385  -0.0013   0.0000   0.9424   0.0007   0.0000   0.2365
   0.0002   0.0000   0.2365   0.0002  -0.0063   0.0000   0.0001  -0.0012  -0.0005   0.7071
   0.0027   0.0005  -0.7070   0.9436  -0.0000   0.0025   0.0748   0.2218   0.0045   0.0748
  -0.2218  -0.0041  -0.0026  -0.0004   0.9962  -0.0002  -0.0008   0.0614  -0.0002  -0.0008
   0.0616
 HESSIAN_MATRIX:MILLIDYNES/ANGSTROM[000045]=
 #  Lower half triangle only
   0.6693  -0.0001   0.4414   0.0000   0.0000   0.0009  -1.3331  -0.8091   0.0000   6.2632
  -0.9734  -0.8792   0.0000   3.5503   3.1702   0.0000   0.0000  -0.0018   0.0000   0.0000
   0.0048  -1.3335   0.8096   0.0000  -0.9515   0.3266   0.0000   6.2648   0.9739  -0.8794
   0.0000  -0.3269   0.3340   0.0000  -3.5523   3.1688   0.0000   0.0000  -0.0018   0.0000
   0.0000   0.0025   0.0000   0.0000   0.0045
 NORMAL_MODE_SYMMETRY_LABELS[0009]=
   1A'     2A'     3A'     1A''    4A'     5A'     2A''    6A'     3A'' 
 VIB._T_DIP:ELECTRONS[0009]=
    0.299    0.203    0.195    1.127    0.620    0.000    0.002    0.617    1.103
 VIB._TRAVEL:ANGSTROMS[0009]= 
    0.152    0.107    0.118    0.000    0.000    0.010    0.000    0.230    0.000
 VIB._RED_MASS:AMU[0009]= 
    0.529    0.523    0.489    0.497    0.489   12.626    0.504   12.689   15.758
 VIB._EFF_MASS:AMU[0009]= 
    1.667    1.516    1.213    0.000    0.006 9999.999    0.001    0.178    0.001
 THERMODYNAMIC_PROPERTIES_TEMPS:K[0000]= 

 ENTHALPY_TOT:CAL/MOL[0000]= 

 HEAT_CAPACITY_TOT:CAL/K/MOL[0000]= 

 ENTROPY_TOT:CAL/K/MOL[0000]= 

 H_O_F(T):KCAL/MOL[0000]= 

 ATOM_X_FORCE:ANGSTROMS[0009]=
   -0.0000   -0.3810    0.0000
    0.7680    0.1800    0.0000
   -0.7679    0.1800    0.0000
 INT_FORCE_CONSTS:MILLIDYNES/ANGSTROM[0009]= 
   10.7082    7.0621    0.0144
    6.3127    3.1953    0.0049
    6.3143    3.1938    0.0045
 CPU_TIME:SECONDS[1]=        0.02
 END OF MOPAC FILE
