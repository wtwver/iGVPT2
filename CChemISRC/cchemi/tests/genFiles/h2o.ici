#RunType=OptFreq
#RunType=Freq
#RunType=GenerateFilesForFreq
#RunType=ComputeFrequenciesFromFiles

#RunType=GenerateFilesOneStepForFreq
#RunType=ComputeFrequenciesOneStepFromFiles

#RunType=GenerateGradFilesForFreq
RunType=ComputeFrequenciesFromGradFiles

#RunType=GenerateGradFilesOneStepForFreq
#RunType=ComputeFrequenciesOneStepFromGradFiles

Model=Mopac
QMKeys=PM3 
dx=0.01
#addH4Corrections=TRUE
#addD3Corrections=TRUE
mopacCommand=mopac
orcaCommand=orca
fireflyCommand=firefly

#Confo
gaussianCommand=g03
fireflyCommand=firefly
numberOfGeometries=3
tolEnergy=0.1
tolDistance=0.1
ConfoOptMM=TRUE
#ConfoOptMopac=TRUE
ConfoOptMopacMethod=PM6 GNORM=0.001
ConfoOptFireFly=FALSE

#Geometry, nAtoms, charge, spin multiplicity. For each atom : symbol, MMType, pdbType, residueName, numResidue, charge, layer, x(Ang),y,z, nconn, num1, type1, num2, type2,...
Geometry
3 0 1
 O OW OW HOH 0 -0.834000 2 1 -0.000010030000 -0.380962440000 0.000000000000 2  2 1 3 1
 H HW HW1 HOH 0 0.417000 2 1 0.767965650000 0.179996340000 0.000000000000 1  1 1
 H HW HW2 HOH 0 0.417000 2 1 -0.767925670000 0.179964470000 0.000000000000 1  1 1

#========================================================================================================================
# Fixed normal modes (ground, n=0) and scaled to T = 300.000000
# Harmonic Velocities
# Velocities, vx(Ang/AKMA-time) vy(Ang/AKMA-time) vz(Ang/AKMA-time): 1fs = 0.020455 AKMA
Velocities
5.886640822088e-02 -2.058223399993e-02 -3.059431322041e-08
-1.171190402840e+00 -1.770315257112e-01 -3.046105812077e-07
2.369377341240e-01 5.036865336133e-01 7.901645607615e-07
#========================================================================================================================
# Fixed normal modes (ground, n=0) and scaled to T = 300.000000
# Harmonic Velocities
# Velocities, vx(Ang/AKMA-time) vy(Ang/AKMA-time) vz(Ang/AKMA-time): 1fs = 0.020455 AKMA
Velocities
-4.016579377963e-02 -3.850457563909e-02 1.100549267935e-07
-4.650631353123e-01 5.367310822391e-01 -1.318023305727e-06
1.102523465640e+00 7.436450921602e-02 -4.286283445034e-07
#========================================================================================================================
#========================================================================================================================
# Fixed normal modes (n=1 for mode -1, n=0 for all other modes) and scaled to T = 300.000000
# Harmonic Velocities
# Velocities, vx(Ang/AKMA-time) vy(Ang/AKMA-time) vz(Ang/AKMA-time): 1fs = 0.020455 AKMA
Velocities
4.144029372186e-02 -9.877512655800e-02 -2.011285370862e-07
-1.377774868056e-01 5.908625988101e-01 1.584291210484e-06
-5.199100836121e-01 9.767704261788e-01 1.607764805956e-06
#========================================================================================================================
