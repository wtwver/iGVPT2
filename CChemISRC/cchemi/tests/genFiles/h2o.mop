PM3 FORCE CHARGE=0 Singlet AUX NOREOR

TEST
O         -0.00001000    -0.38096200     0.00000000  
H          0.76796600     0.17999600     0.00000000  
H         -0.76792600     0.17996400     0.00000000  
 
