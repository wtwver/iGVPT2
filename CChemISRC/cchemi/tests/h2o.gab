[Gabedit Format]
[GEOCONV]
energy
8.516833
max-force
0.000000
rms-force
0.000000

[GEOMETRIES]
3

 H 0.60000000 -0.44453900 -0.19577200
 H 0.10000000 0.97655000 -0.71769400
 O 0.60000000 0.46799000 0.09323100

[GEOMS] 1
1 3
energy kcal/mol 1
deltaE K 1
Dipole Debye 3
8.516833
0
1.01581140275189 0.76432208442977 -1.88949698675065
3 0 1
 H HW HW1 HOH 1 0.417000 2 1 0.60000000 -0.44453900 -0.19577200 1  3 1
 H HW HW2 HOH 1 0.417000 2 1 0.10000000 0.97655000 -0.71769400 1  3 1
 O OW OW HOH 1 -0.134000 2 1 0.60000000 0.46799000 0.09323100 2  1 1 2 1
