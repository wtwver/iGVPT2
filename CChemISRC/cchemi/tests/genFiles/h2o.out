 *******************************************************************************
 ** Site#: 1079         For non-commercial use only     Version 15.146L 64BITS**
 *******************************************************************************
 ** Cite this work as:                                                        **
 **                                                                           **
 ** (a) MOPAC2012, James J. P. Stewart, Stewart Computational Chemistry,      **
 **  Version 15.146L web: HTTP://OpenMOPAC.net         Days remaining: 166    **
 **                                                                           **
 **                                 AND                                       **
 **                                                                           **
 ** (b) Maia, J.D.C., et al., J. Chem. Theory Comput. 2012, 8, 3072-3081.     **
 **                DOI: 10.1021/ct3004645.                                    **
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2012                                  **
 **                                                                           **
 *******************************************************************************

                            PM3 CALCULATION RESULTS


 *******************************************************************************
 *  CALCULATION DONE:                                Fri Dec 11 08:34:06 2015  *
 *  PM3        - The PM3 Hamiltonian to be used
 *  SINGLET    - SPIN STATE DEFINED AS A SINGLET
 *  NOREOR     - DO NOT REORIENTATE SYSTEM WHEN WORKING OUT THE SYMMETRY
 *
 *
 *
 *                 CHARGE ON SYSTEM =     0
 *
 *
 *
 *  FORCE      - FORCE CALCULATION SPECIFIED
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY  7200.000 SECONDS
 *  AUX        - OUTPUT AUXILIARY INFORMATION
 *******************************************************************************
PM3 FORCE CHARGE=0 Singlet AUX NOREOR

 TEST
   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       O         -0.00001000  *  -0.38096200  *   0.00000000  *
     2       H          0.76796600  *   0.17999600  *   0.00000000  *
     3       H         -0.76792600  *   0.17996400  *   0.00000000  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM         X         Y         Z

     1         O       -0.0000   -0.3810    0.0000
     2         H        0.7680    0.1800    0.0000
     3         H       -0.7679    0.1800    0.0000

  H: (PM3): J. J. P. STEWART, J. COMP. CHEM. 10, 209 (1989).
  O: (PM3): J. J. P. STEWART, J. COMP. CHEM. 10, 209 (1989).

           Empirical Formula: H2 O  =     3 atoms



      MOLECULAR POINT GROUP   :   Cs  


 SINGLET STATE CALCULATION


      RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS =    4



          HEAT OF FORMATION =     -53.432825 KCALS/MOLE


          INTERNAL COORDINATE DERIVATIVES

   NUMBER  ATOM       BOND         ANGLE      DIHEDRAL

     1       O     -0.064123    -0.028552     0.000000
     2       H      0.109498     0.037730     0.000000
     3       H     -0.045375    -0.009178     0.000000
 DIPOLE           X         Y         Z       TOTAL
 POINT-CHG.     0.000     0.966     0.000     0.966
 HYBRID        -0.000     0.772    -0.000     0.772
 SUM            0.000     1.739    -0.000     1.739


          GRADIENT NORM =   0.14312

          MOLECULAR WEIGHT =   18.02



          ROTATIONAL CONSTANTS IN CM(-1)

          A =   29.92599445   B =   14.18037598   C =    9.62132791



           PRINCIPAL MOMENTS OF INERTIA IN UNITS OF 10**(-40)*GRAM-CM**2

          A =        0.9354   B =        1.9741   C =        2.9095


          ORIENTATION OF MOLECULE IN FORCE CALCULATION

    NO.       ATOM         X         Y         Z

     1          O      -0.0000   -0.3810    0.0000
     2          H       0.7680    0.1800    0.0000
     3          H      -0.7679    0.1800    0.0000


    FIRST DERIVATIVES WILL BE USED IN THE CALCULATION OF SECOND DERIVATIVES

    FOR POINT-GROUP Cs   THERE ARE   2 UNIQUE SYMMETRY FUNCTIONS.

 STEP:   1 TIME =     0.00 SECS, INTEGRAL =      0.00 TIME LEFT: 172800.00
 STEP:   2 TIME =     0.00 SECS, INTEGRAL =      0.00 TIME LEFT: 172800.00
 STEP:   3 TIME =     0.00 SECS, INTEGRAL =      0.00 TIME LEFT: 172800.00
 STEP:   4 TIME =     0.00 SECS, INTEGRAL =      0.00 TIME LEFT: 172800.00
 STEP:   5 TIME =     0.00 SECS, INTEGRAL =      0.00 TIME LEFT: 172800.00
 STEP:   6 TIME =     0.00 SECS, INTEGRAL =      0.00 TIME LEFT: 172800.00
 STEP:   7 TIME =     0.00 SECS, INTEGRAL =      0.00 TIME LEFT: 172800.00
 STEP:   8 TIME =     0.00 SECS, INTEGRAL =      0.01 TIME LEFT: 172799.99
 STEP:   9 TIME =     0.00 SECS, INTEGRAL =      0.01 TIME LEFT: 172799.99

(To convert to Hartree/Bohr^2, multiply by (10^(-21) x N x a0^2)/(4.184 x 627.51) =  0.0642)


           FORCE MATRIX IN MILLIDYNES/ANGSTROM


                  O  1       H  2       H  3
 ------------------------------------------
     O    1  12.826802
     H    2   8.182890   8.698725
     H    3   8.186125   1.118067   8.701556


          HEAT OF FORMATION =     -53.432825 KCALS/MOLE

          ZERO POINT ENERGY      13.721 KCAL/MOL


           NORMAL COORDINATE ANALYSIS (Total motion = 1 Angstrom)


   Root No.       1         2         3

                 1 A'      2 A'      3 A'  

                1742.0    3867.8    3988.4
  
           1   -0.0000   -0.0484   -0.0001
           2   -0.0498   -0.0001    0.0312
           3    0.0002   -0.0001   -0.0001
           4   -0.2642    0.3852   -0.4156
           5    0.3949    0.2812   -0.2468
           6   -0.0022    0.0000   -0.0002
           7    0.2642    0.3832    0.4173
           8    0.3949   -0.2801   -0.2481
           9   -0.0008    0.0021    0.0021


           MASS-WEIGHTED COORDINATE ANALYSIS (NORMAL COORDINATES)


   Root No.       1         2         3

                 1 A'      2 A'      3 A'  

                1742.0    3867.8    3988.4
  
           1   -0.0000   -0.2755   -0.0006
           2   -0.2830   -0.0004    0.1784
           3    0.0011   -0.0007   -0.0007
           4   -0.3771    0.5503   -0.5969
           5    0.5637    0.4018   -0.3544
           6   -0.0031    0.0000   -0.0003
           7    0.3771    0.5475    0.5994
           8    0.5637   -0.4001   -0.3563
           9   -0.0011    0.0030    0.0030


          DESCRIPTION OF VIBRATIONS


 VIBRATION    1    1A'      ATOM PAIR        ENERGY CONTRIBUTION    RADIAL
 FREQ.      1741.97        O  1 --  H  2          +50.0% ( 54.5%)     4.4%
 T-DIPOLE    0.2992        O  1 --  H  3          +50.0% ( 54.5%)     4.4%
 TRAVEL      0.1524
 RED. MASS   0.5291
 EFF. MASS   1.6670

 VIBRATION    2    2A'      ATOM PAIR        ENERGY CONTRIBUTION    RADIAL
 FREQ.      3867.83        O  1 --  H  2          +50.2% ( 72.6%)    96.9%
 T-DIPOLE    0.2034        O  1 --  H  3          +49.8% ( 72.3%)    96.8%
 TRAVEL      0.1072
 RED. MASS   0.5226
 EFF. MASS   1.5164

 VIBRATION    3    3A'      ATOM PAIR        ENERGY CONTRIBUTION    RADIAL
 FREQ.      3988.42        O  1 --  H  3          +50.2% ( 69.1%)    99.1%
 T-DIPOLE    0.1954        O  1 --  H  2          +49.8% ( 68.7%)    99.0%
 TRAVEL      0.1181
 RED. MASS   0.4886
 EFF. MASS   1.2128

           FORCE CONSTANT IN CARTESIAN COORDINATES (Millidynes/A)

    ATOM   CHEMICAL       X               Y               Z
   NUMBER   SYMBOL  FORCE CONSTANT  FORCE CONSTANT  FORCE CONSTANT

      1       O       10.708246        7.062075        0.014396
      2       H        6.312668        3.195273        0.004851
      3       H        6.314293        3.193825        0.004526

          JOB ENDED NORMALLY



 TOTAL JOB TIME:             0.02 SECONDS

 == MOPAC DONE ==
